docs/reference/child_admin.rst

```diff
@@ -18,15 +18,23 @@ its parent:
 
 .. configuration-block::
 
+    .. code-block:: yaml
+
+        # app/config/services.yml
+        app.admin.playlist:
+            class: App\Admin\PlaylistAdmin
+            calls:
+                - [ addChild, ['@app.admin.video', 'video']]
+
     .. code-block:: xml
 
-        <!-- app/config/config.xml -->
-        <service id="sonata.admin.playlist" class="App\Admin\PlaylistAdmin">
+        <!-- app/config/services.xml -->
+        <service id="app.admin.playlist" class="App\Admin\PlaylistAdmin">
             <!-- ... -->
 
             <call method="addChild">
-                <argument type="service" id="sonata.admin.video" />
-                <argument>playlist</argument>
+                <argument type="service" id="app.admin.video" />
+                <argument>video</argument>
```

Comment:
I am not sure, but I guess it should be the name of the child like this? 🤔

Reply:
Nop, you need to provide parent field on your child entity. So if Video is the entity it should have `playlist` field that points to Playlist Entity.

Reply:
But I am calling `addChild` on the Playlist, and add 'Video' to it.

So the second Parameter is `Name of the Parent` instead of `Name of the Child`? (@see #5109 )

Reply:
ah, my bad there, I rechecked the code a few minutes ago and then I see it is the name of the parent or in this case the value you had in your `$parentAssociationMapping`

Reply:
Ok 👍